README.md

```diff
@@ -0,0 +1,57 @@
+# Restaurant Javascript Application
+
+## The application is created as a part requirement of the Microverse fullstack developer course.
+
+#### In this application, I show cased the use of webpack and JavaScript modules. I also show cased import and export of modules.
+
+### Built With
+
+- Javascript
+- Visual Studio Code
+- Webpack
+
+### System dependencies
+
+- web Browser
+
+### Get project up and running
+
+```
+git clone https://github.com/chrystalme/RestaurantApp.git
+```
+
+```
+cd RestaurantApp
+```
+
+```
+npm install
+```
+
+## Live Link
+
+## Authors
+
+👤 **Malieze Afam Ifeanyi**
+
+- GitHub: [@chrystalme](https://github.com/chrystalme)
+- Twitter: [@afam_ifeanyi](https://twitter.com/afam_ifeanyi)
+- LinkedIn: [Malieze Afam Ifeanyi(chrys)](https://linkedin.com/afam-chrys)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/chrystalme/RestaurantApp/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Microverse](https://microverse.com)
```

Comment:
[optional] This link is broken, I think it should be `https://microverse.org` instead of ~`https://microverse.com`~